persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -122,6 +125,21 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor {
     case GetState =>
       sender() ! state
 
+    case GetStatePaginated(page) =>
+      val tsErrState = state.tsParsingErrors
+      val tsErrorCount = if (tsErrState.isEmpty) 0 else 1
+
+      val (tsErrorsReturn, offset) =
+        if (tsErrState.nonEmpty && page == 1) (tsErrState, 1)
+        else (Seq(), 0)
+
+      val totalLarErrors = state.larParsingErrors.size
+      val from = Math.min(totalLarErrors, (page - 1) * 20)
```

Comment:
`20` is the number of results per page, right? If so, would it make sense to make this configurable?